engine/src/main/java/org/hibernate/search/bridge/spi/FieldType.java

```diff
@@ -48,5 +48,10 @@
 	/**
 	 * A double field.
 	 */
-	DOUBLE
+	DOUBLE,
+
+	/**
+	 * A container for othe fields, for example an embedded object or a map.
```

Comment:
In Elasticsearch you cannot set dynamic:true on a field that it's not an object (or nested). FieldType
does not have a way to define a similar type. At the moment FIeldTYpe allows only primitive types.

I guess the javadoc shouldn't talk about containers, I'll update it